src/main/java/com/_up/megastore/controllers/interfaces/IUserController.java

```diff
@@ -15,4 +13,10 @@ public interface IUserController {
   void activateUser(@PathVariable UUID userId,
       @RequestBody @Valid ActivateUserRequest activateUserRequest);
 
+  @PostMapping("/recover-password/send-email")
+  void sendEmailToRecoverPassword(@RequestParam String email);
+
+  @PostMapping("/{userId}/recover-password")
+  void recoverPassword(@PathVariable UUID userId, @RequestParam String newPassword, @RequestParam UUID recoverPasswordToken);
```

Comment:
Los parametros de las contraseñas deberian ir por body no como request param, deberias crear un DTO "RecoverPasswordRequest" que contenga ambos parametros como atributos

```suggestion
  void recoverPassword(@PathVariable UUID userId, @RequestBody RecoverPasswordRequest recoverdPasswordRequest);
```

Además deberias agregar la repetición de la nueva contraseña